[2022-04-15 13:52:43,651] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 scheduled__2020-01-18T00:00:00+00:00 [queued]>
[2022-04-15 13:52:44,483] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 scheduled__2020-01-18T00:00:00+00:00 [queued]>
[2022-04-15 13:52:44,555] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-15 13:52:44,555] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-04-15 13:52:44,555] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-15 13:52:45,493] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): task_4> on 2020-01-18 00:00:00+00:00
[2022-04-15 13:52:45,618] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', 'scheduled__2020-01-18T00:00:00+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpxkxajepn', '--error-file', '/tmp/tmpmocjpx8w']
[2022-04-15 13:52:45,725] {standard_task_runner.py:52} INFO - Started process 117171 to run task
[2022-04-15 13:52:46,256] {standard_task_runner.py:80} INFO - Job 112: Subtask task_4
[2022-04-15 13:52:50,519] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_4 scheduled__2020-01-18T00:00:00+00:00 [running]> on host 10.0.2.15
[2022-04-15 13:52:55,127] {taskinstance.py:1446} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=solfa
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2020-01-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-18T00:00:00+00:00
[2022-04-15 13:52:55,229] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-15 13:52:55,311] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-04-15 13:52:55,698] {subprocess.py:85} INFO - Output:
[2022-04-15 13:52:59,139] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-15 13:53:01,362] {taskinstance.py:1278} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20200118T000000, start_date=20220415T135243, end_date=20220415T135301
[2022-04-15 13:53:02,546] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-15 13:53:03,903] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-15 13:53:05,085] {dagrun.py:545} INFO - Marking run <DagRun parallel_dag @ 2020-01-18 00:00:00+00:00: scheduled__2020-01-18T00:00:00+00:00, externally triggered: False> successful
[2022-04-15 13:53:05,283] {dagrun.py:590} INFO - DagRun Finished: dag_id=parallel_dag, execution_date=2020-01-18 00:00:00+00:00, run_id=scheduled__2020-01-18T00:00:00+00:00, run_start_date=2022-04-15 13:50:38.876116+00:00, run_end_date=2022-04-15 13:53:05.283144+00:00, run_duration=146.407028, state=success, external_trigger=False, run_type=scheduled, data_interval_start=2020-01-18 00:00:00+00:00, data_interval_end=2020-01-19 00:00:00+00:00, dag_hash=2e1728272660f48fa2de34c79560b2dd
